modules/apps/account/account-service/src/main/java/com/liferay/account/internal/model/listener/ObjectEntryModelListener.java

```diff
@@ -116,9 +125,8 @@ private void _validateAccountEntry(
 
 		throw new ObjectDefinitionAccountEntryRestrictedException(
 			StringBundler.concat(
-				"The account entry ", accountEntryId,
-				" does not exist or the user ", userId,
-				" does not belong to it"));
+				"The user ", userId, " does not belong to the account ",
```

Comment:
I believe separating these messages leaks information, since you're indirectly telling a malicious user that an account in fact exists and they guessed the ID correctly.

Reply:
@rodrigopaulino that makes sense. I will revert back to the original message. Thanks for pointing that out.